src/config/style_edition.rs

```diff
@@ -30,7 +30,8 @@ macro_rules! style_edition_default {
                     $crate::config::StyleEdition::Edition2015
                     | $crate::config::StyleEdition::Edition2018
                     | $crate::config::StyleEdition::Edition2021 => $default_2015,
-                    $crate::config::StyleEdition::Edition2024 => $default_2024,
+                    $crate::config::StyleEdition::Edition2024
+                    | $crate::config::StyleEdition::Edition2027 => $default_2024,
```

Comment:
Edition2024 default will also extend to 2027. Is this the desired behaviour?

Reply:
For now I think that's fine since we don't currently have plans to change any default option values for `2027`, but it would be nice to add a TODO comment explaining that we'll need to update the macro once we need to start changing defaults for 2027.

Reply:
@johnhuichen want to remind you about the ask to add a comment here.